RSTAB/LoadCasesAndCombinations/resultCombination.py

```diff
@@ -25,6 +41,47 @@ def __init__(self,
         # Result Combination No.
         clientObject.no = no
 
+        # Result Combination Name
+        if name:
+            clientObject.user_defined_name_enabled = True
+            clientObject.name = name
+
+        # Assigned Design Situation
+        clientObject.design_situation = design_situation
+
+        # Result Combination Type
+        clientObject.combination_type = combination_type.name
+
+        # Combination Items
+        if combination_items:
+            clientObject.items = model.clientModel.factory.create('ns0:result_combination.items')
+            for i,j in enumerate(combination_items):
+                rci = model.clientModel.factory.create('ns0:result_combination_items_row')
+                rci.no = i+1
+                rci.row = model.clientModel.factory.create('ns0:result_combination_items')
+                clearAttributes(rci.row)
+                rci.row.case_object_item = combination_items[i][0]
+                rci.row.operator_type = combination_items[i][1].name
+                rci.row.case_object_factor = combination_items[i][2]
+                rci.row.case_object_load_type = combination_items[i][3].name
```

Comment:
If parenthesis also available in RSTAB like RFEM then you may update in RSTAB too like in RFEM. 
https://github.com/Dlubal-Software/RFEM_Python_Client/pull/292/files

Reply:
Addressed